components/omega/test/CMakeLists.txt

```diff
@@ -118,6 +118,42 @@ target_link_libraries(${_TestLoggingName} ${OMEGA_LIB_NAME} yakl)
 add_test(NAME LOGGING_TEST COMMAND ./${_TestLoggingName})
 
 
+##################
+# Config test
+##################
+
+set(_TestConfigName testConfig.exe)
+
+# Add broadcast test
+add_executable(${_TestConfigName} infra/ConfigTest.cpp)
+
+target_include_directories(
+  ${_TestConfigName}
+  PRIVATE
+  ${OMEGA_SOURCE_DIR}/src/base
+  ${OMEGA_SOURCE_DIR}/src/infra
+)
+
+target_compile_options(
+  ${_TestConfigName}
+  PRIVATE
+  ${OMEGA_CXX_FLAGS}
+)
+
+target_link_options(
+  ${_TestConfigName}
+  PRIVATE
+  ${OMEGA_LINK_OPTIONS}
+)
+
+target_link_libraries(${_TestConfigName} ${OMEGA_LIB_NAME} yaml-cpp)
+
+add_test(
+  NAME CONFIG_TEST
```

Comment:
I believe `CONFIG_TEST` needs to be added to the list in `set_tests_properties` below.